webcompat/static/js/lib/wizard/step2.js

```diff
@@ -0,0 +1,72 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/* Allows the user to select a category of the problem they're experiencing */
+
+import utils from "./utils.js";
+import notify from "./notify.js";
+
+const container = $(".step-container.step2");
+const detectionBug = $("#problem_category-0");
+const siteBug = $("#problem_category-1");
+const layoutBug = $("#problem_category-2");
+const videoBug = $("#problem_category-3");
+const unknownBug = $("#problem_category-4");
+const sitePrettyUrl = $("#website-url");
+const otherProblem = $(".other-problem");
+const nextStepButton = $(".next-step.step-2");
```

Comment:
There's a similar comment here about numbering here in the html, and we see it that we conveniently translate it in JS as what is it really.

`layoutBug` is more telling than `#problem_category-2`. :) HTML may need a bit of rewriting.